Makefile

```diff
@@ -1127,14 +1127,13 @@ ocamllex.opt: ocamlopt
 lex/ocamllex$(EXE): OC_BYTECODE_LINKFLAGS += -compat-32
 
 partialclean::
-	rm -f lex/*.cm* lex/*.o lex/*.obj
+	rm -f lex/*.cm* lex/*.o lex/*.obj \
+        $(ocamllex_PROGRAMS) $(ocamllex_PROGRAMS:=.exe) \
```

Comment:
The $(ocamllex_PROGRAMS) (and .exe) line is new (it was not part of the removed `clean` fragment below), why is it here?

Reply:
Ah, in spotting that bit after, the original fix in the commit has become less clear! It's two changes - there's the movement of the those lines from the `clean` target below (the title of the PR) and there's also the _addition_ of the cleaning of the programs (the title of the commit). The original line (from bb375e) was:
https://github.com/ocaml/ocaml/blob/bb375e46fffa91b3152c8f75997c77e0df6b0f33/lex/Makefile#L57-L58
which got omitted.